clouddriver-ecs/src/main/java/com/netflix/spinnakerclouddriver/aws/ecs/EcsCloudProvider.java

```diff
@@ -0,0 +1,34 @@
+package com.netflix.spinnakerclouddriver.aws.ecs;
+
+import org.springframework.stereotype.Component;
+import com.netflix.spinnaker.clouddriver.core.CloudProvider;
+
+import java.lang.annotation.Annotation;
+
+//@Component
```

Comment:
My thinking was that ECS would appear as a cloud provider if @component is not commented out, and might give false hopes.  Perhaps that assertion of mine was incorrect, and leaving @component in would not mislead anyone?